src/test/java/org/folio/eusage/reports/MainVerticleTest.java

```diff
@@ -59,7 +59,7 @@ static JsonObject getCounterReportMock(UUID id, int cnt) {
     report.put("customer", new JsonArray()
         .add(new JsonObject()
             .put("id", "fake customer id")
-            .put("reportItems", new JsonArray()
+            .put(cnt > 1 ? "reportItems" : "Report_Items", new JsonArray()
```

Comment:
Seriously? The name of the JSON field varies in capitalisation and use of underscores dependent on _how many_ records there are??

Reply:
This is just mock where I want to return both types in one response...

Reply:
Ah, that's a relief!